spec.html

```diff
@@ -4847,95 +4847,22 @@ <h1>
       </emu-clause>
     </emu-clause>
 
-    <emu-clause id="sec-toboolean" type="abstract operation">
+    <emu-clause id="sec-toboolean" oldids="table-toboolean-conversions" type="abstract operation">
       <h1>
         ToBoolean (
           _argument_: an ECMAScript language value,
         ): a Boolean
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It converts _argument_ to a value of type Boolean according to <emu-xref href="#table-toboolean-conversions"></emu-xref>:</dd>
+        <dd>It converts _argument_ to a value of type Boolean.</dd>
       </dl>
-      <emu-table id="table-toboolean-conversions" caption="ToBoolean Conversions" oldids="table-10">
-        <table>
-          <tr>
-            <th>
-              Argument Type
-            </th>
-            <th>
-              Result
-            </th>
-          </tr>
-          <tr>
-            <td>
-              Undefined
-            </td>
-            <td>
-              Return *false*.
-            </td>
-          </tr>
-          <tr>
-            <td>
-              Null
-            </td>
-            <td>
-              Return *false*.
-            </td>
-          </tr>
-          <tr>
-            <td>
-              Boolean
-            </td>
-            <td>
-              Return _argument_.
-            </td>
-          </tr>
-          <tr>
-            <td>
-              Number
-            </td>
-            <td>
-              If _argument_ is *+0*<sub>𝔽</sub>, *-0*<sub>𝔽</sub>, or *NaN*, return *false*; otherwise return *true*.
-            </td>
-          </tr>
-          <tr>
-            <td>
-              String
-            </td>
-            <td>
-              If _argument_ is the empty String (its length is 0), return *false*; otherwise return *true*.
-            </td>
-          </tr>
-          <tr>
-            <td>
-              Symbol
-            </td>
-            <td>
-              Return *true*.
-            </td>
-          </tr>
-          <tr>
-            <td>
-              BigInt
-            </td>
-            <td>
-              If _argument_ is *0*<sub>ℤ</sub>, return *false*; otherwise return *true*.
-            </td>
-          </tr>
-          <tr>
-            <td>
-              Object
-            </td>
-            <td>
-              Return *true*.
-              <emu-note>
-                <p>An alternate algorithm related to the [[IsHTMLDDA]] internal slot is mandated in section <emu-xref href="#sec-IsHTMLDDA-internal-slot-to-boolean"></emu-xref>.</p>
-              </emu-note>
-            </td>
-          </tr>
-        </table>
-      </emu-table>
+      <emu-alg>
+        1. If _argument_ is a Boolean, return _argument_.
+        1. If _argument_ is any of *undefined*, *null*, *+0*<sub>𝔽</sub>, *-0*<sub>𝔽</sub>, *NaN*, *0*<sub>ℤ</sub>, or the empty String, return *false*.
```

Comment:
PR #2877 proposes "any of" used this way, but the current spec doesn't use it. Instead, the current spec would say:
- `_argument_ is *undefined*, ...`
- `_argument_ is either *undefined*, ...`
- `_argument_ is one of *undefined*, ...`

Reply:
I'm okay with this. It's informal language.

Reply:
> It's informal language.

It is, but that's true of most of the spec. Every "consistent phrasing" PR changes informal language. So it seems like "informal language" can't be a reason to not aim for consistency. So why bring it up here?

Now, if you were to say "Yeah, it's an anomaly, but #2877 will take care of it soon, so it's not worth fixing here," I'd find that less puzzling.

Reply:
@jmdyck I am aiming for consistency, but am okay with more inconsistency for the time being (in informal language) if it's heading in the direction we want to go, as here.